packages/UI/src/components/progressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,54 @@
+import * as Progress from '@radix-ui/react-progress';
+import React from 'react';
+
+interface ProgressProps {
+  /** The starting color of the progress bar. */
+  startColor: string;
+  /** The color of the progress bar when progress is complete. */
+  endColor: string;
+  /** The background color of the progress bar container. */
+  backgroundColor: string;
+  /** The width of the progress bar. Can be any TailwindCSS valid value. */
+  width: string;
+  /**
+   * The height of the progress bar. Can be any TailwindCSS valid value.
+   */
+  height: string;
+  /**
+   * Progress value passed as a prop
+   */
+  progress: number;
+}
+/** Defines the properties for the ProgressBar component */
+const ProgressBar: React.FC<ProgressProps> = ({
+  startColor,
+  endColor,
+  backgroundColor,
+  width,
+  height,
```

Comment:
these props aren't needed as they're all going to be fixed

Reply:
Ok perfect!